pyquil/tests/test_device.py

```diff
@@ -74,11 +148,25 @@ def test_kraus_model():
     assert KrausModel.from_dict(d) == km
 
 
-def test_noise_model():
-    km1 = KrausModel('I', (5.,), (0, 1), [np.array([[1 + 1j]])], 1.0)
-    km2 = KrausModel('RX', (np.pi / 2,), (0,), [np.array([[1 + 1j]])], 1.0)
-    nm = NoiseModel("my_qpu", [km1, km2], {0: np.eye(2), 1: np.eye(2)})
+def test_noise_model(kraus_model_I, kraus_model_RX90):
+    km1 = kraus_model_I
+    km2 = kraus_model_RX90
+    nm = NoiseModel(DEVICE_FIXTURE_NAME, [km1, km2], {0: np.eye(2), 1: np.eye(2)})
 
     assert nm == NoiseModel.from_dict(nm.to_dict())
     assert nm.gates_by_name("I") == [km1]
     assert nm.gates_by_name("RX") == [km2]
+
+
+def test_device(device_fixture, isa, noise_model):
```

Comment:
i think this test is a bit difficult to follow because of the hierarchy of fixtures used as inputs. it tests relationships between objects that are entirely defined outside of the scope of this test. i think the best way forward for this test in particular would be to drop the `device_fixture` and `device_raw` fixtures and instead build the device object from its constituent parts inside of the UT